downstream/modules/aap-on-azure/con-aap-azure-about.adoc

```diff
@@ -0,0 +1,28 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="con-aap-azure-about"]
+
+= About {AAPonAzureName}
+
+[role="_abstract"]
+
+{AAPonAzureName} is a managed application that you can deploy from the link:https://azure.microsoft.com/en-us/marketplace/[Azure Marketplace] portal to a subscription in your Azure tenant.
+{AAPonAzureNameShort} provides access to a library of Ansible content collections, and it is integrated with key Azure services, so you can start deploying, configuring, and managing infrastructure and applications quickly.
+
+The following Red Hat Automation Platform components are available on Ansible Automation Platform for Azure:
+
+* Automation Controller
+* Automation Hub
+* Private Automation Hub
+* Automation Service Catalog
+* Ansible Content Collections, including the Microsoft collection for Azure
+* Automation Execution Environment
+* Ansible content tools, including access to Red Hat Insights for the Ansible Automation Platform
+
+Automation mesh is not available on Ansible Automation Platform for Azure.
```

Comment:
Could this be a note, so that it isn't missed?

Reply:
Added note.